google/cloud/spanner_v1/types/transaction.py

```diff
@@ -29,9 +29,306 @@
 
 
 class TransactionOptions(proto.Message):
-    r"""TransactionOptions are used to specify different types of transactions.
-
-    For more info, see: https://cloud.google.com/spanner/docs/reference/rest/v1/Transaction
+    r"""Transactions
+    ============
```

Comment:
@larkee Could you edit the source protos to remove the headings? https://github.com/googleapis/googleapis/blob/d76f26f335b2b570b7e19d938f35f9894236e565/google/spanner/v1/transaction.proto#L31

> Any formatting in comments must be in CommonMark. Headings and tables must not be used, as these cause problems for several tools, and are unsuitable for client library reference documentation.

https://google.aip.dev/192#formatting